src/bin.ts

```diff
@@ -37,6 +38,7 @@ const tsNodeFlags = {
 
 const tsNodeAlias = {
   'transpile-only': 'T',
+  'swc': 'W',
```

Comment:
from what I can see in the official docs, ts-node does not use `-W` short flag for swc, there's only a full `--swc` option